File: backend/src/main/java/com/backend/backend/entity/Room.java
package com.backend.backend.entity;

import com.baomidou.mybatisplus.annotation.*;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("room")
public class Room {

    @TableId(type = IdType.AUTO)
    private Long id;

    private Long cinemaId; // 所属影院ID

    private String name; // 影厅名称 (如 1号厅)

    @JsonProperty("rows")
    private Integer rowsCount; // 座位行数

    @JsonProperty("cols")
    private Integer colsCount; // 座位列数

    private String seatTemplate; // 座位模板 (JSON 格式)

    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;

    @TableField(fill = FieldFill.INSERT_UPDATE)
    private LocalDateTime updateTime;
} 

File: frontend/src/types/room.ts
/**
 * 影厅信息接口
 */
export interface Room {
  id: number
  cinemaId: number
  name: string
  rows: number // 行数
  cols: number // 列数
  seatTemplate?: string // 座位模板 (可选)
  createTime?: string // 创建时间 (ISO 8601 string)
  updateTime?: string // 更新时间 (ISO 8601 string)
}
